packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -39,22 +39,30 @@ export interface Props extends Themeable {
   minValue: number;
   orientation: VizOrientation;
   itemSpacing?: number;
-  displayMode: 'basic' | 'lcd' | 'gradient';
+  cellWidth?: number;
```

Comment:
cellWidth is a non-obvious name/meaning for BarGauge.

If it needs to be an external property, can we add comments about what it is supposed to do?

Reply:
yea, can rename to lcdCellWidth